client/src/SimpleSnackbar.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Snackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle'
+import {makeStyles, Theme} from '@material-ui/core/styles';
+
+
+const useStyles=makeStyles((theme:Theme)=>({
+    root:{
+        width: '100%',
+    },
+}))
+
+export default function SimpleSnackbar(){
+    const classes=useStyles()
+    const [open, setOpen]=React.useState(true)
```

Comment:
conventional to prefix with an `is` or a `b` (for boolean) for boolean vars
because otherwise an `open` prop could be misconstrued as a function (open is also a verb)
also do the typing thing here too `useState<boolean>(`